### PR TITLE
Review section about BadgeView control.

### PR DESCRIPTION
**Pull request by mariaterzieva12**

### CODE REVIEW

File: controls/badgeview/badge-overview.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ slug: badge-overview
 
 The Badge is the indicator (or marker) which is displayed to show notifications, statuses, and so on, while the BadgeView is the control in which you can add content and position the Badge based on this content.
 
->important To provide the best experience out of the BadgeView control and the Badge indicator, it is recommended that you use the [BadgeView]({%slug badgeview-overview%}) and customize the Badge whenever needed.
+>important To provide the best experience out of the BadgeView control and the Badge indicator, it's recommended that you use the [BadgeView]({%slug badgeview-overview%}) and customize the Badge whenever needed.
```
Comment (yordan-mitev) [inline controls/badgeview/badge-overview.md:13]: I find this note very confusing. Can you rephrase and clarify it?
It sounds like "For the best experience with the Badge and BadgeView, customize the Badge and use the BageView," which is nonsense.

File: controls/badgeview/configuration.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ The following image shows the final result.
 
 ![BadgeView Badge Text](images/badgeview-badgetext.png)
 
->tip You can fully customize the look and feel of the BadgeView. For detailed information, refer to articles on [BadgeView styling]({%slug badgeview-styling%}) and [customization]({%slug badgeview-customization%}).
+>tip You can customize the look and feel of the BadgeView. For detailed information, refer to articles on [BadgeView styling]({%slug badgeview-styling%}) and [customization]({%slug badgeview-customization%}).
```
Comment (yordan-mitev) [inline controls/badgeview/configuration.md:65]:
```suggestion
>tip You can customize the look and feel of the BadgeView. For detailed information, refer to the articles on [BadgeView styling]({%slug badgeview-styling%}) and [customization]({%slug badgeview-customization%}).
```

File: controls/badgeview/getting-started.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ var badge = new RadBadgeView();
 xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
  ````
 
-1. Set `Content` and `BadgeText` properties. Note that the Badge marker (indicator), which is part of the BadgeView, will be visualized only if the `Content` property of the BadgeView is set.
+1. Set `Content` and `BadgeText` properties. Note that the Badge marker (indicator), which is part of the BadgeView, will be visualized only if set the `Content` property of the BadgeView.
```
Comment (yordan-mitev) [inline controls/badgeview/getting-started.md:44]:
```suggestion
1. Set the `Content` and `BadgeText` properties. Note that the Badge marker (indicator), which is part of the BadgeView, will be visualized only if you set the `Content` property of the BadgeView.
```

**Review by yordan-mitev — COMMENTED** [review]
In all descriptions, change the component name to "Telerik UI for .NET MAUI BadgeView".

File: controls/badgeview/overview.md [hunk 4]
```diff
@@ -20,10 +20,10 @@ Badges can be used as an additional marker for any element: to decorate avatars,
...
+* [Alignment]({%slug badgeview-position-alignment%}#badge-alignment)&mdash;You can align the Badge indicator according to its content.
 * [Animation]({%slug badgeview-animation%})&mdash;You can choose whether to have an `Animation` while displaying the Badge indicator. In addition, you can specify the animation duration and animation easing.
 * [Badge types]({%slug badgeview-predefined-badges%})&mdash;You can select the type of a Badge from a set of various predefined options. Also, you can customize and style each Badge type.
-* [Flexible styling API]({%slug badgeview-styling%}) and [customization options]({%slug badgeview-customization%})–Allows you to change the `BackgroundColor`, `BorderThickness`, and `Color` properties of the BadgeView. Also, the BadgeView exposes an API which you can use to fully customize and style the Badge indicator.
+* [Flexible styling API]({%slug badgeview-styling%}) and [customization options]({%slug badgeview-customization%})–Allows you to change the `BackgroundColor`, `BorderThickness`, and `Color` properties of the BadgeView. Also, the BadgeView exposes an API which you can use to customize and style the Badge indicator.
```
Comment (yordan-mitev) [inline controls/badgeview/overview.md:26]:
```suggestion
* Flexible [styling API]({%slug badgeview-styling%}) and [customization options]({%slug badgeview-customization%})–Allows you to change the `BackgroundColor`, `BorderThickness`, and `Color` properties of the BadgeView. Also, the BadgeView exposes an API, which you can use to customize and style the Badge indicator.
```

File: controls/badgeview/badge-overview.md (hunk 1 above)
Comment (didiyordanova) [reply controls/badgeview/badge-overview.md]: description was confusing, i have changed it

**Review by didiyordanova — APPROVED** [review]
